gt/main.c

```diff
@@ -631,6 +730,16 @@ alloc_gt_conf(void)
 static inline void
 cleanup_gt_instance(struct gt_instance *instance)
 {
+	if (instance->neigh.cache_tbl != NULL) {
```

Comment:
1. Why aren't we calling `destroy_neigh_hash_table()` here instead of having an ad-hoc code?

2. Aren't we forgetting to release `instance->neigh6`?